deploy/ftlib-test.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ftlib-test
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: ftlib-test-app
+  template:
+    metadata:
+      labels:
+        app: ftlib-test-app
+    spec:
+      containers:
+        - image: ftlib:deploy
+          imagePullPolicy: Never
+          name: ftlib-test-c
+          ports:
+          - containerPort: 3000
+            name: http
+          - containerPort: 22
+            name: ssh
+          command: ["python3", "/ftlib/test/generic_test.py"]
```

Comment:
Will this be sufficient for the deploy test? But honestly, I still cannot figure out a way to test, specially given we are building something 'fault-tolerant'.

Reply:
Good call. I updated this to pytorch test. We haven't added the validation logic around this yet. Testing the fault-tolerant part can be a bit tricky that requires some thoughts.